rocket_packet.c

```diff
@@ -69,17 +69,20 @@ unsigned int serialize_avionics_data(AvionicsData* data, uint8_t* dst) {
 unsigned int serialize_command_packet(CommandPacket* pkt, uint8_t* dst) {
 	unsigned int offset = 0;
 	
-	memcpy(dst + offset, (void *) &pkt->start_char, 2);
-	offset += 2;
+	memcpy(dst + offset, (void *) &pkt->start_short, sizeof(pkt->start_short));
+	offset += sizeof(pkt->start_short);
```

Comment:
good shit les sizeofs, j'avais écris ça dans le temps que je savais pas que c'était une macro et non une fonction faque je pensais être optimal

Reply:
I read one of the functions in the document and saw that some places had a sizeof and others did not, so I replaced the hardcoded values.